dev/custom-pages/social.scss

```diff
@@ -241,55 +241,26 @@ $color-linkedin: #0077b5;
   }
 }
 
-.dcwss.dc-wall {
-  .stream {
-    li {
-      background: #393939;
-
-      .inner {
-        color: #FFFFFF;
-      }
-
-      .section-thumb {
-        float: none;
-        margin: 0 auto;
-
-        img {
-          margin: 7px auto;
-          max-width: 100%;
-        }
-      }
-
-      .section-intro {
-        padding: 20px 15px 20px 35px;
-
-        a { 
-          border-bottom: 0 !important;
-
-          &:hover {
-            border-bottom: 0 !important;
-          }
-        }
-      }
-
-      .section-text {
-        a {
-          color: #fff !important;
+.crt-social-icon {
+  .crt-icon-facebook {
+    color: $color-facebook!important;
+  }
 
-          &:hover {
-            border-bottom: none !important;
-            text-decoration: underline !important;
-          }
-        }
-      }
+  .crt-icon-youtube {
+    color: $color-youtube!important;
+  }
 
-      .socicon {
-        bottom: 17px;
-      }
-    }
+  .crt-icon-instagram {
+    background: #f09433;
+    background: -moz-linear-gradient(45deg, #f09433 0%, #e6683c 25%, #dc2743 50%, #cc2366 75%, #bc1888 100%);
```

Comment:
Vendor prefixes aren't needed here--they will be added by autoprefixer if necessary